debian/copy_installer.sh

```diff
@@ -3,6 +3,12 @@ then
    echo "Copy python-saithriftv2.install as python-saithrift.install"
    cp ./debian/installerFiles/python-saithriftv2.install ./debian/python-saithrift.install
 else
-   echo "Copy python-saithriftv1.install as python-saithrift.install"
-   cp ./debian/installerFiles/python-saithriftv1.install ./debian/python-saithrift.install
+   if test -n "lsb_release -a | grep bullseye"
```

Comment:
How about the bookworm, debian 12?
Suggest to check if buster, then do something.